### DISCUSSION TITLE
Include empty lines in flashcards

### DISCUSSION THREAD

**Discussion by Maano26** [reuseman/flashcards-obsidian#78] [Q&A]
Hey everyone, 

is there any way to include empty lines into the flashcards? 

For example 

> This is the question #card
> This line is included
> 
> This line is currently not included 
> But I would love include it as well

Usually the answers to my flashcards a quite long and including empty lines massivley improves their readibility. If empty lines cannot be included, is there some workaround to improve readability of long cards? 

Best!

**Comment by reuseman** [comment]
Hey you can just use a space character 

```md
This is the question #card
This line is included
 <- here there is the space character
This line is currently not included
But I would love include it as well
```

**Reply by danburonline** [reply]
Nice, but when you lint/auto-format your markdown files, this won't work.
